app/models/incoming_message.rb

```diff
@@ -55,7 +55,8 @@ class IncomingMessage < ActiveRecord::Base
   # never really has many info_request_events, but could in theory
   has_many :info_request_events, :dependent => :destroy
 
-  belongs_to :raw_email, :dependent => :destroy
```

Comment:
Yeah; that fixed the issue mentioned in the changelog (https://github.com/rails/rails/issues/12380).